wordpress-mobile/WordPress-Android
# Add label with the current app language to the App Settings screen

Fixes #16483

Shows a label with the current app language in App Settings, under "Interface Language".

To test:
1. Open the WordPress or Jetpack app of this PR (QR-code or local build)
2. Tap on the profile icon to open the `Me` page
3. Tap `App Settings`
4. **Expect** a label with the current language to show under "Interface Language" (in the first list item)
   Optionally:
   - Tap `Interface Language` and select choose another language
   - **Expect** the label to update and show the new language that you've selected.

Preview:
| English (Day Mode) | Dutch (Night Mode) |
| --- | --- |
| ![currentLang](https://user-images.githubusercontent.com/4588074/182906407-7d2a06eb-2559-4cd4-be48-6787ab8e65db.png) | ![currentLangNl](https://user-images.githubusercontent.com/4588074/182906423-3b8cb7f7-7e8f-4abb-85b9-10232efa625d.png) |

## Regression Notes
1. Potential unintended areas of impact
   Language Picker.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
   Tested manually.

7. What automated tests I added (or what prevented me from doing so)
   Updated `LocalePickerViewModelTest` unit tests.

PR submission checklist:
